kmod/core/core.c

```diff
@@ -314,27 +257,57 @@ static const struct stacktrace_ops kpatch_print_trace_ops = {
  */
 static int kpatch_verify_activeness_safety(struct kpatch_module *kpmod)
 {
+
+#define MAX_STACK_TRACE_DEPTH   64
+
 	struct task_struct *g, *t;
+	unsigned long *entries;
+	unsigned long nr_entries;
+	int i;
 	int ret = 0;
 
-	struct kpatch_backtrace_args args = {
-		.kpmod = kpmod,
-		.ret = 0
-	};
+	entries = kmalloc(MAX_STACK_TRACE_DEPTH * sizeof(*entries), GFP_KERNEL);
+	if (!entries)
+		return -ENOMEM;
```

Comment:
I think sleeping isn't allowed in stop_machine(), so this should probably be allocated on the stack.